card/lib/card/query/attributes.rb

```diff
@@ -25,31 +25,31 @@ def right val
       end
 
       def editor_of val
-        acts_alias, actions_alias =
-          "a#{table_id force=true}", "an#{table_id force=true}"
-        joins << Join.new(
+        act_join = Join.new(
           from: self,
-          to: ['card_acts', acts_alias, 'actor_id' ]
+          to: ['card_acts', "a#{table_id force=true}", 'actor_id' ]
```

Comment:
Useless assignment to variable - `force`.<br>Surrounding space missing for operator '='.<br>Space inside square brackets detected.